### DISCUSSION TITLE
S21.x版本开启导出header后出现横向和纵向的滚动条是因为什么？

### DISCUSSION THREAD

**Discussion by Zhaohanqi12** [antvis/S2#2716] [General]
![image](https://github.com/antvis/S2/assets/166707652/0d522926-0b63-4285-b3f9-72bbf3bf482f)

**Comment by lijinke666** [comment]
具体需要复现链接才能排查, 一张图看不出来为什么, 应该是开启 header 后 `width: 100%`, 把表格外部容器撑满了, 可以试试给外部容器 overflow: hidden
